client/src/redux/show-cert-saga.js

```diff
@@ -1,4 +1,3 @@
-import { navigate } from 'gatsby';
```

Comment:
gotta put this back

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient